cmd/root.go

```diff
@@ -168,9 +174,19 @@ func validateAuthentication(cmd *cobra.Command, args []string) error {
 }
 
 func ExecuteContext(ctx context.Context) error {
```

Comment:
is this called everytime at the end of a command if I understand correctly?

Reply:
https://github.com/groundcover-com/cli/blob/ab5d3c73cfbb16a98e20dc83c461e5b1e080b732/main.go#L44

called as the main process